src/server/examples/examples_test.go

```diff
@@ -73,3 +73,184 @@ func TestExampleTensorFlow(t *testing.T) {
 	require.NoError(t, c.DeleteRepo("GoT_train", false))
 	require.NoError(t, c.DeleteRepo("GoT_scripts", false))
 }
+
+func TestWordCount(t *testing.T) {
+
+	if testing.Short() {
+		t.Skip("Skipping integration tests in short mode")
+	}
+
+	c := getPachClient(t)
+
+	// Should stay in sync with doc/examples/word_count/README.md
```

Comment:
I could read it, but I'd have to parse out the JSON. Additionally, I do actually need to change the pipeline a bit (to point it at HN not wikipedia).

We discussed adding another layer of checks to the test in the future where it behaves similar to the `assets.go` test. It would check that the JSON strings used here match what we see in this file, and otherwise fails the test. We'd have to handle the case where the JSON deviates on purpose (probably parse the JSON in the test, change the field, emit the JSON, then compare?).

This is a bit tricky. So for now, this seems adequate.